healthcare/api-client/v1/consent/attribute_definitions.py

```diff
@@ -0,0 +1,494 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+
+# [START healthcare_create_resource_attribute_definition]
+def create_resource_attribute_definition(
+    project_id: str,
+    location: str,
+    dataset_id: str,
+    consent_store_id: str,
+    resource_attribute_definition_id: str,
+):
+    """Creates a RESOURCE attribute definition. A RESOURCE attribute is an attribute whose value is
+    determined by the properties of the data or action.
+
+    See https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/healthcare/api-client/v1/consent
+    before running the sample."""
+    # Imports the Google API Discovery Service.
+    from googleapiclient import discovery
+
+    api_version = "v1"
+    service_name = "healthcare"
+    # Returns an authorized API client by discovering the Healthcare API
+    # and using GOOGLE_APPLICATION_CREDENTIALS environment variable.
+    client = discovery.build(service_name, api_version)
+
+    # TODO(developer): Uncomment these lines and replace with your values.
+    # project_id = 'my-project'  # replace with your GCP project ID
+    # location = 'us-central1'  # replace with the parent dataset's location
+    # dataset_id = 'my-dataset'  # replace with the FHIR store's parent dataset ID
+    # consent_store_id = 'my-consent-store'  # replace with the consent store's ID
+    # resource_attribute_definition_id = 'requester_identity'  # replace with the attribute definition ID
+    consent_store_parent = (
+        "projects/{}/locations/{}/datasets/{}/consentStores/{}".format(
+            project_id, location, dataset_id, consent_store_id
+        )
+    )
+
+    body = {
+        "description": "whether the data is identifiable",
+        "category": "RESOURCE",
+        "allowed_values": ["identifiable", "de-identified"],
+    }
+
+    request = (
+        client.projects()
+        .locations()
+        .datasets()
+        .consentStores()
+        .attributeDefinitions()
+        .create(
+            parent=consent_store_parent,
+            body=body,
+            attributeDefinitionId=resource_attribute_definition_id,
+        )
+    )
+
+    response = request.execute()
+    print("Created RESOURCE attribute definition: {}".format(response))
+
+    return response
+
+
+# [END healthcare_create_resource_attribute_definition]
+
+
+# [START healthcare_create_resource_attribute_definition]
```

Comment:
Should this be `create_request`? It has the same region tag as the function above.

Reply:
Yes, thanks for catching.